internal/controllers/suite_test.go

```diff
@@ -94,10 +94,10 @@ var _ = AfterSuite(func() {
 })
 
 func namesFromList(list client.ObjectList) []string {
-	var names []string
-
 	items, err := meta.ExtractList(list)
 	Expect(err).NotTo(HaveOccurred())
+
+	names := make([]string, 0, len(items))
```

Comment:
Preallocation of the slice. There are few other places like this

https://golangci-lint.run/usage/linters/#prealloc